### DISCUSSION TITLE
What about the x coordinate scale of partial plots？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#178] [Q&A]
**What about the x coordinate scale of partial plots？**
library(randomForestSRC)
airq.obj <- rfsrc(Ozone ~ ., data = airquality)
plot.variable(airq.obj, xvar.names = "Wind", partial = TRUE)  #partial plots

![image](https://user-images.githubusercontent.com/5112397/149515977-09f63153-f146-4e2e-9533-27155e67d6fd.png)

**Comment by ishwaran** [comment]
Those are "tick marks" which show the values of the actual data.

**Reply by A-Pai** [reply]
I see，thanks

**Comment by A-Pai** [comment]
I see
